src/rules/at-mixin-no-risky-parent-selectors/README.md

```diff
@@ -0,0 +1,142 @@
+# mixin-no-risky-parent-selectors
+
+If a mixin contains a parent selector within another style rule, and is used in a nested context, 
```

Comment:
If we really want to comply with Stylelint's guides, then this "single-line description" should be in the form of "Disallow ...". https://stylelint.io/developer-guide/rules/#write-the-readme

Reply:
There are examples in stylelint's repo:
https://github.com/stylelint/stylelint/tree/main/lib/rules/at-rule-no-vendor-prefix